chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +332,76 @@ int ChromeNetworkDelegate::OnBeforeURLRequest(
   return rv;
 }
 
+std::string generateBlockedElemhideJavaScript(std::string url, std::string relative_url) {
+  std::stringstream ss;
+
+  // basically copy-pasted JS and saved to resources from
+  // https://github.com/adblockplus/adblockpluschrome/blob/master/include.preload.js#L546
+  std::string elemhide_js =
+    ui::ResourceBundle::GetSharedInstance()
+    .GetRawDataResource(IDR_ADBLOCK_ELEMHIDE_JS)
+    .as_string();
+  ss << elemhide_js;
+
+  ss <<
```

Comment:
i'm feeling strange about this part.
We can use Grit **messages** and insert values on-the-fly, but we will need to format JS as XML and it's undesired. Another option is to store it as file (like JS from `core` above) with our own tokens like `%url%` and replace tokens to actual values - looks a bit not perfect too.

Leave as-is for now.